activation/activation.go

```diff
@@ -54,6 +55,7 @@ func DefaultPoetConfig() PoetConfig {
 		RequestRetryDelay:     400 * time.Millisecond,
 		MaxRequestRetries:     10,
 		CertifierInfoCacheTTL: 5 * time.Minute,
+		PowParamsCacheTTL:     5 * time.Minute,
```

Comment:
Would it make sense to make this just one parameter instead of 2?

Reply:
Maybe :shrug:. Giving extra flexibility doesn't hurt and it is useful in systests where certifier doesn't change at all (and can be cached for long time) but the pow_params change every epoch (1 minute)